src/pages/offer-page/index.tsx

```diff
@@ -37,7 +36,7 @@ function OfferPage({ offers }: OfferPageProps): JSX.Element {
   } = currentOffer;
 
   const ratingWidth = `${(rating / 5) * 100}%`;
-  const nearPlaces = offers.filter((offer) => offer.id !== offerId).slice(0, 3);
+  const nearPlaces = allOffers.filter((offer) => offer.id !== offerId).slice(0, 3);
```

Comment:
If this is repetitive functionality I'd rather move it to utilities. Otherwise you are good